### DISCUSSION TITLE
Parameters for the 8k model?

### DISCUSSION THREAD

**Discussion by LearnedVector** [snakers4/silero-vad#76] [Q&A] [answered]
## Parameters for the 8k model?

Are the default samples tune for the 8k models? or should i half the sample size for things like `num_samples_per_window`, `min_speech_samples`, etc?

**Comment by snakers4** [comment, marked as answer]
Hi,

The default params are definitely for 16k

If you are experimenting with 8k, please treat the params as a blank slate - plot some charts, see how it works, try different combinations
